dev-util/mh/files/Makefile.patch

```diff
@@ -0,0 +1,21 @@
+diff --git a/Makefile b/Makefile
+index 01023ca..f87ab56 100644
+--- a/Makefile
++++ b/Makefile
+@@ -14,6 +14,7 @@ endif
+ 
+ DEBUG = -g -fsanitize=address
+ CFLAGS = -Wall -lpcre2-8 #$(DEBUG)
```

Comment:
1. wrong, `-lpcre2-8` should be in `LIBS`
2. what about user `CFLAGS`?

Reply:
Fixed in the Makefile in version 1.0.3.